### PR TITLE
fix broken wording and add .isEmpty and .contains methods

### PR DESCRIPTION
**Pull request by IgorSamurovic**
I also think we should add .isEmpty() to all data structures, what do you think?

### CODE REVIEW

File: wurst/data/HashList.wurst [hunk 1]
```diff
@@ -24,27 +23,37 @@ public class HashList<T>
...
 			if t castTo int == ht.loadInt(this castTo int, i)
 				removeAt(i)
 				break
-	
+
 	/** Get the size of the list */
 	function getSize() returns int
 		return size
 
+	/** Return whether the list is empty */
```
Comment (Cokemonkey11) [inline wurst/data/HashList.wurst]: What's the value of this comment?

File: wurst/data/HashList.wurst [hunk 2]
```diff
@@ -24,27 +23,37 @@ public class HashList<T>
...
+	function isEmpty() returns bool
+		return size == 0
+
 	/** Get the element at the given index from this list */
 	function get(int index) returns T
 		return ht.loadInt(this castTo int, index) castTo T
 
+	/** Return whether the element exists in the list */
```
Comment (Cokemonkey11) [inline wurst/data/HashList.wurst]: How are elements compared? (by value)

References are compared for equality.

**Comment by Cokemonkey11** [comment]
Looks good overall.

**Comment by Frotty** [comment]
good, thanks! 🌴

**Pull request merged**